docs/book/src/cronjob-tutorial/api-design.md

```diff
@@ -17,7 +17,7 @@ machines.  You've probably noticed them when specifying resources requests
 and limits on pods in Kubernetes.
 
 They conceptually work similar to floating point numbers: they have
-a significand, base, and exponent.  Their serialize, human readable for
+a significand, base, and exponent.  Their serialized human readable format
```

Comment:
Maybe we should use `Their serializable and human readable format`

Reply:
sure, I wasn't sure what the original intent for the wording here was.

Reply:
@mengqiy updated